README.md

```diff
@@ -15,10 +15,45 @@ adds the following new functionality:
   abstractions](https://crossplane.io/docs/master/concepts.html), enabling
   Alibaba Cloud resources to fulfill a user's general need for cloud services
 
-## Getting Started and Documentation
+## Getting Started
 
-For getting started guides, installation, deployment, and administration, see
-our [Documentation](https://crossplane.io/docs/latest).
+In the following, We will deploy flight tracker app using OAM and Alibaba RDS resource:
+
+![alt text](./doc/img/flight-tracker.png)
+
+Make sure you have installed:
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+- kubectl
+- [helm](https://helm.sh/)
+
+Then set up the environment on first trial:
+
+```bash
+helm repo add crossplane-master https://charts.crossplane.io/master/
+git clone git@github.com:crossplane/addon-oam-kubernetes-local.git ../addon-oam-kubernetes-local
+export ACCESS_KEY_ID=<your-access-key-id>
+export ACCESS_KEY_SECRET=<your-access-key-secret>
+```
+
+Access key should be obtained via Alibaba clooud.
+
+Then run command:
+```bash
+make demo
```

Comment:
This command seems too easy, readers won't understand what real happens. I suggest we explain step by step and take the chance to tell what real happen behind these instructions.

Reply:
Good suggestion. I add a line to explain the general ideas of the command.

Usually getting started is to get users up and running without getting into the details. Users won't read so much either. So the general part should be good enough.